ios/flutter_blue.podspec

```diff
@@ -23,7 +23,7 @@ Bluetooth Low Energy plugin for Flutter.
     ss.source_files = "gen/*.pbobjc.{h,m}", "gen/**/*.pbobjc.{h,m}"
     ss.header_mappings_dir = "gen"
     ss.requires_arc = false
-    ss.dependency "Protobuf", '~> 3.11.4'
```

Comment:
Did the generated protobuf file change with this version change?

Reply:
Sorry, I didn't mean to include the last two commits (protobuf related) in this PR. Should I delete them, or have you already cherry picked them onto your [fork](https://github.com/CsabaConsulting/flutter_blue/commit/159f6f4cfd0d145daf642b0927bd3856164b68ce)?

Reply:
I'm good, no worries.

Reply:
I like your PR BTW. I saw that particular version change in other PRs

Reply:
I have an own fork, which has multiple PRs merged into it. Pretty sad how stale is flutter_blue.